corehq/apps/geospatial/forms.py

```diff
@@ -20,20 +18,14 @@ class GeospatialConfigForm(forms.ModelForm):
     class Meta:
         model = GeoConfig
         fields = [
-            "location_data_source",
             "user_location_property_name",
             "case_location_property_name"
         ]
 
-    location_data_source = forms.CharField(
```

Comment:
Removed this field until we support location geo-location data. For now the only possible setting is fetching location data from the custom user data.